package-lock.json

```diff
@@ -0,0 +1,6527 @@
+{
```

Comment:
Should be this file part of a GIT repository? isn't possible it will be tested with different versions than on the newly created project?